remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -189,7 +189,7 @@ public void deleteCategoryById(final StoreRef storeRef, final String id, final P
     @Override
     public List<Category> listCategoriesForNode(final String nodeId, final Parameters parameters)
     {
-        final NodeRef contentNodeRef = nodes.validateNode(nodeId);
+        final NodeRef contentNodeRef = nodes.validateOrLookupNode(nodeId, null);
```

Comment:
I assume that the reason to change the method to retrieve the `NodeRef` throughout the whole PR might be that it handles getting `NodeRef` more properly but passing `null` as a parameter doesn't look great (not only here - I know we do that in other places too). Maybe we should refactor `validateNode` method a bit instead - but that's jus  an idea - if that could be a risky effort - then just disregard this comment?